.github/workflows/main.yml

```diff
@@ -34,15 +25,20 @@ jobs:
           curl -sH "Accept: application/vnd.github.v3+json" \
           https://api.github.com/repos/ocaml/ocaml/commits/trunk \
           | jq .commit.tree.sha | xargs printf '::set-output name=commit::%s'
+
       - name: Use OCaml ${{ matrix.ocaml-compiler }}
         uses: ocaml/setup-ocaml@v2
         with:
-          ocaml-compiler: ${{ matrix.ocaml-compiler }}
+          opam-pin: false
+          opam-depext: false
+          ocaml-compiler: ocaml.5.0.0,ocaml-option-mingw
           opam-repositories: |
-            default: https://github.com/ocaml/opam-repository.git
-            alpha: https://github.com/kit-ty-kate/opam-alpha-repository.git
+            local: ${{ github.workspace }}\.github\opam\custom
+            dra27: https://github.com/dra27/opam-repository.git#windows-5.0
+            override: https://github.com/shym/custom-opam-repository.git
+            default: https://github.com/ocaml-opam/opam-repository-mingw.git#sunset
+            upstream: https://github.com/ocaml/opam-repository.git
```

Comment:
I don't think you need:
- `local` - as that refers to custom opam files in [the `multicoretests` repo](https://github.com/ocaml-multicore/multicoretests/tree/main/.github/opam) that you don't have here
- `override` - as that refers to @shym's [repo of packages patched for 5.1 and `trunk`](https://github.com/shym/custom-opam-repository/tree/master/packages) which is not tested here

A quick test without those two lines should tell us... :crossed_fingers:

Reply:
Thanks !